addons/lunch/static/src/xml/lunch.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates>
+    <t t-name="lunch.PreviousOrder">
+        <div t-if="widget.pref_ids == 0" name="preferences">
+            <div class="oe_inline oe_lunch_intro">
+                <h3>This is the first time you order a meal</h3>
+                <p class="oe_grey">Select a product and put your order comments on the note.</p>
+                <p class="oe_grey">Your favorite meals will be created based on your last orders.</p>
+                <p class="oe_grey">Don't forget the alerts displayed in the reddish area</p>
```

Comment:
text-muted is giving much darker shade but here i think we needed something lighter than that thats why 'oe_grey' of base.css used.

![notes](https://cloud.githubusercontent.com/assets/5112424/5625622/c15f2c2e-95a4-11e4-9908-6078bcf8a317.png)